apluslms_roman/backends/docker.py

```diff
@@ -31,6 +32,8 @@ def create_container(client, **opts):
             logger.warning("Failed to stop container %s: %s", container, err)
 
 
+
```

Comment:
Why so much space?

Reply:
hm forgot it there after moving the function away, I guess